scripts/presets/travis/replica_sets/replicaset.json

```diff
@@ -12,8 +12,7 @@
                 "nssize": 1,
                 "port": 3000,
                 "bind_ip_all": true,
-                "smallfiles": true,
-                "setParameter": {"enableTestCommands": 1}
+                "smallfiles": true
```

Comment:
Why are you removing `enableTestCommand`?

I'm curious if this is the reason you needed to introduce `skip_if_sleep_command_unavailable()`. Perhaps there's a bug in `skip_if_test_commands_disabled()`?